documentation/docs/03-loading.md

```diff
@@ -58,14 +58,16 @@ type LoadOutput = {
 
 If `load` returns nothing, SvelteKit will [fall through](#routing-advanced-fallthrough-routes) to other routes until something responds, or will respond with a generic 404.
 
-> `load` only applies to components that define pages, not the components that they import.
->
-> It is important to note that `load` may run on either the server or in the client browser. Code called inside `load` blocks:
->
-> - should use the SvelteKit-provided [`fetch`](#loading-input-fetch) method for getting data in order to avoid duplicate network requests
-> - should generally run on the same domain as any upstream API servers requiring credentials
-> - should not reference `window`, `document`, or any browser-specific objects
-> - should not reference any API keys or secrets directly, which will be exposed to the client, but instead call an endpoint using any required secrets
+> `load` only applies to [pages](#routing-pages) and [layouts](#layouts), and it may run on either the server or in the client browser.
```

Comment:
```suggestion
> `load` only applies to [page](#routing-pages) and [layout](#layouts) components, not the components that they import, and by default runs on both the server and the client browser.
```

Reply:
> and by default runs on both the server and the client browser

Not quite sure about this bit, it can only run on either server or client but not really both at once. I think the previous one already describes it pretty nicely.

Reply:
It doesn't run on both at the same exact instant, but it does run on both for a single request

Reply:
```suggestion
> `load` only applies to [page](#routing-pages) and [layout](#layouts) components, and can run both on the server (unless [ssr](#ssr-and-javascript-ssr) is disabled) and in the browser.
```